docs/api/storage/psa_protected_storage.md

```diff
@@ -0,0 +1,17 @@
+## PSA protected storage
+
+PSA protected storage APIs enable saving data to and retrieving data from PSA protected storage.
+
+Unlike [PSA internal storage](../apis/psa_internal_storage.html), PSA protected storage always runs in the Non-Secure Processing Environment (NSPE) and redirects calls to the KVStore static API.
+
+<span class="notes">**Note:** In general, we recommend using the [KVStore static API](../storage/KVStoreGlobalAPI.html) in the NSPE.</span>
+
+### PSA protected storage class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](../mbed-os-api-doxy/protected__storage_8h.html)
+
```

Comment:
This will need an example.

Reply:
Actually, ignore that. Because we want to discourage this, we don't want an example here.